src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -602,6 +660,57 @@ ad_utility::streams::stream_generator ExportQueryExecutionTrees::
   co_yield "\n</sparql>";
 }
 
+// _____________________________________________________________________________
+template <>
+ad_utility::streams::stream_generator ExportQueryExecutionTrees::
+    selectQueryResultToStream<ad_utility::MediaType::sparqlJson>(
+        const QueryExecutionTree& qet,
+        const parsedQuery::SelectClause& selectClause,
+        LimitOffsetClause limitAndOffset,
+        CancellationHandle cancellationHandle) {
```

Comment:
After putting some thought into this,
Maybe the cleaner way is to give
all those functions the `[[maybe_unused]]` request timer,
s.t. the `qleverJson` export can just be a `normal ` template specialization here.

Reply:
This is fine now as it is.